### DISCUSSION TITLE
Project removal and restore policy per v1.0

### DISCUSSION THREAD

**Discussion by Yaevh** [rasmus/EventFlow#885] [Q&A]
I'm trying to get some work done in #417, starting from `develop-v1` branch, but `EventFlow.MsSql` project was removed from the solution in #810 (the same happened to pretty much every other project except core `EventFlow`). What was the reason for this decision?

I can still add `EventFlow.MsSql` manually and work from there, but then I encounter two problems:
1. Should I re-add it to solution on pull request? What's the recommended policy?
2. There's a mismatch between different versions of NUnit - `EventFlow.MsSql.Tests` uses 3.12.0, while `EventFlow.TestHelpers` uses 3.13.2. Should I upgrade NUnit in `EventFlow.MsSql.Tests` and commit (though it's not a part of the .sln at this point), or should I fix it only locally and keep the main repo as is?

So basically, what's the policy on supporting projects other than core `EventFlow`?

**Comment by rasmus** [comment]
Hi @Yaevh

Sorry for the confusion. All projects that doesn't yet compile in v1 has been removed in order to simplify the conversion and release process. Basically projects are updated one-by-one and added to the solution as they are updated. The updates are mainly focused on replacing `ILog` with `ILogger<>` and `IResolver` with `IServiceProvider`.

 You can view the current state of projects in the [versions](https://github.com/eventflow/EventFlow#versions) section of the readme.

**Comment by rasmus** [comment]
`EventFlow.MsSql` has since been added to the solution and released as part of an alpha version for v1
